msteams-platform/apps-in-teams-meetings/Test Preview for Monetized apps.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Test Preview for Monetized apps 
+author: v-ypalikila
+description: Create and test SaaS Preview offers for Teams app before pushing the offer live.
+ms.topic: conceptual
+ms.author: v-ypalikila
+ms.localizationpriority: medium
+keywords: teams apps SaaS offer preview offer  app source preview
+---
+
+# Test Preview for Monetized apps
+
+As a developer, you can create a Software as a Service (SaaS) offer and test the end-to-end purchase experience of a Teams app before pushing the offer live. Only users who have been added as the preview audience for the Teams app can test the preview offer.
+
+## Prerequisites
+
+1. [Developer Account](/office/developer-program/microsoft-365-developer-program-get-started)
+1. [Create a SaaS offer](/azure/marketplace/create-saas-dev-test-offer)
+1. [Add a preview audience for a SaaS offer](/azure/marketplace/create-new-saas-offer-preview)
+
+## Create a preview offer ID
+
+You need to create a preview offer ID for the SaaS offer using the **App Source preview link** in the Partner Center.
+
+1. Go to [Partner Center](https://go.microsoft.com/fwlink/?linkid=2166002) and sign in using your developer credentials.
+1. Select **Marketplace offers**.
+1. Select the SaaS offer you want to preview.
+1. Select **App source preview link** under **Go Live** to generate a Preview Offer ID.
+1. Copy the Preview Offer ID generated in the URL along with the *-Preview* suffix.
+
+:::image type="content" source="../assets/images/apps-in-meetings/publish-status-publisher-signoff.png" alt-text="App source preview":::
+
+## Configure your app with the Preview offer ID
+
+After you've generated your Preview offer ID, you must link the offer ID to your Teams app for the audience to view your Preview offer in the Teams Store.
+
+1. Go to [Developer Portal](https://dev.teams.microsoft.com/).
+1. Select **Apps** from the left nav pane.
```

Comment:
As per MSTP Nav pane or navigation pane is acceptable. Make the change accordingly